python/example/single_cell_cable.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python3
+
+import arbor
+import argparse
+import numpy as np
+
+import pandas
+import seaborn  # You may have to pip install these.
+
+
+class Cable(arbor.recipe):
+    def __init__(self, probes,
+                 Vm, length, radius, cm, rL, g,
+                 stimulus_start, stimulus_duration, stimulus_amplitude,
+                 cv_policy_max_extent):
+        """
+        probes -- list of probes
+
+        Vm -- membrane leak potential
+        length -- length of cable in micrometer
+        radius -- radius of cable in micrometer
+        cm -- membrane capacitance in F/m^2
+        rL -- axial resistivity in Ohm*centimeter
+        g -- membrane conductivity in Siemens/cm^2
+
+        stimulus_start -- start of stimulus in ms
+        stimulus_duration -- duration of stimulus in ms
+        stimulus_amplitude -- amplitude of stimulus in nA
+
+        cv_policy_max_extent -- maximum extent of control volume in micrometer
+        """
+
+        # (4.1) The base C++ class constructor must be called first, to ensure that
+        # all memory in the C++ class is initialized correctly.
+        arbor.recipe.__init__(self)
+
+        self.the_probes = probes
+
+        self.Vm = Vm
+        self.length = length
+        self.radius = radius
+        self.cm = cm
+        self.rL = rL
+        self.g = g
+
+        self.cv_policy_max_extent = cv_policy_max_extent
+
+        self.the_props = arbor.neuron_cable_properties()
+        self.the_cat = arbor.default_catalogue()
+        self.the_props.register(self.the_cat)
+
+    def num_cells(self):
+        return 1
+
+    def num_sources(self, gid):
+        assert gid == 0
+        return 0
+
+    def cell_kind(self, gid):
+        assert gid == 0
+        return arbor.cell_kind.cable
+
+    def cell_description(self, gid):
```

Comment:
I think this method could benefit from some comments, because it'll show up in the tutorial.